sortinghat/exceptions.py

```diff
@@ -90,7 +90,14 @@ class DatabaseError(BaseError):
     """Exception raised when a database error occurs"""
 
     message = "%(error)s (err: %(code)s)"
-    code = CODE_DATABASE_ERROR
+    code = CODE_DATABASE_EXISTS
+
+
+class DatabaseExists(BaseError):
```

Comment:
I miss test cases for this exception. Check `test_exceptions.py` file. Test are straightforward.